src/utils/command.rs

```diff
@@ -43,6 +44,12 @@ pub fn run_command(command: Command) -> Result<ActionResult, ActionError> {
     match std::process::Command::new(&command.name)
         .envs(&command.env)
         .args(&command.args)
+        .current_dir(
+            &command
+                .dir
+                .clone()
+                .unwrap_or(std::env::current_dir().unwrap().into_os_string().into_string().unwrap()),
```

Comment:
I think that the only thing missing now is this. I had to `.unwrap().into_os_string().into_string().unwrap()` because `current_dir()` gives us a PathBuf. But I think I am screwing something up here in the process. It seems that command is always executing in `std::env::current_dir()` anyways

Reply:
I'll take a look 👍

Reply:
It looks fine. What output do you see when you run it? Sorry, I don't have an Arch machine atm

Reply:
Yeah, this was not the problem. It seems that if we try to use `require_root: true` for users that have `NOPASSWD: ALL` nothing breaks but commands are not executed. Same thing for running on shell that recently executed sudo stuff.

Reply:
Tried this on a machine that I have with ubuntu to check the apt provider, and even though comtrya runs fine, in the end packages are not installed. Not sure why `require_root: true` is making it silently fail.

Reply:
Hmm. I'll dig in to that. Thanks for letting me know

Reply:
I'm going to add some end to end tests to the CI job to ensure this doesn't happen again, and I'll add assertion support to manifest 👍

Reply:
Awesome!